### ISSUE TITLE
Unable to locate srt-devel and srt-libs packages during installation

### ISSUE THREAD

**Issue by Funkvay** [pkeroulas/st2110-toolkit#4]
Hi Patrick.

I tried to install the transcoder software on my Ubuntu system by running `./install.sh transcoder`, but I encountered the following errors:
```
E: Unable to locate package srt-devel
E: Unable to locate package srt-libs
```

I searched for these packages in the Ubuntu package repositories using the apt-cache search command, but I couldn't find them. I also tried to download the packages manually from trusted sources, but I couldn't find any suitable sources for these packages.

As a result, I was unable to complete the installation of the transcoder software on my Ubuntu system. Can you please advise on how I can resolve this issue and install the transcoder software successfully?

**Comment by pkeroulas** [comment]
Hi, this is due to a wrong OS-related logic that should now [be fixed](https://github.com/pkeroulas/st2110-toolkit/commit/7cf2f697952f6c2300b1e781de60e264c2ff7d57).
Let me know.

**Comment by Funkvay** [comment]
yeah, that solved my problem and it works now, thank you!
 I have one more question, If I need to use ffmpeg, I need to use also `./install` that is in transcoder directory? It's written not to use that script, but there are no ffmpeg binaries that I can use.
 I just want to use ffmeg to connect SMPTE 2110 with my C++ code

**Comment by pkeroulas** [comment]
Going back to to your `ffmpeg` question make sure, it is correctly built. The ouput should look like:
```
ffmpeg version n5.1-6-gffaf606130 Copyright (c) 2000-2022 the FFmpeg developers
  built with gcc 9 (Ubuntu 9.4.0-1ubuntu1~20.04.1)
  configuration: --prefix=/usr/local --extra-cflags=-I/usr/local/include --extra-ldflags=-L/usr/local/lib --bindir=/usr/local/bin --extra-libs=-ldl --enable-version3 --enable-gpl --enable-nonfree --enable-postproc --enable-libsrt --enable-libx264 --enable-libfdk-aac --enable-libmp3lame --disable-ffplay --disable-ffprobe --enable-small --disable-stripping --disable-debug
```

**Comment by pkeroulas** [comment]
For `gstreamer`, make sure you installed `gst-plugins-good`. I haven't used gstreamer for years.
Note that ffmpeg now can generate ST 2110 streams but I haven't tested it yet.

**Comment by funkvay-star** [comment]
yeah I saw about `ffmpeg` and st 2110. Trying to find a way how to use it. About building I got this 
`st2110-toolkit/transcoder/install.sh: line 147: ffmpeg_gpu_options: unbound variable`
So yeah, sadly it's still not build correctly.

for `gstreamer` I downloaded all needed, but I'm still not sure whats the problem, that's why I'm going to try build it from source.

**Comment by matiaspl** [comment]
> For `gstreamer`, make sure you installed `gst-plugins-good`. I haven't used gstreamer for years. Note that ffmpeg now can generate ST 2110 streams but I haven't tested it yet.

Could you elaborate how it could be done? Just using RTP muxer?
